src/server/middleware/auth.ts

```diff
@@ -248,6 +248,15 @@ const handleWebsocketAuth = async (
     // Set as a header for `getUsers` to parse the token.
     req.headers.authorization = `Bearer ${jwt}`;
     const user = await getUser(req);
+
+    const isIpInAllowlist = await checkIpInAllowlist(req);
+    if (!isIpInAllowlist) {
+      return {
+        isAuthed: false,
+        error: "Unauthorized IP address",
```

Comment:
We can make the message more helpful by adding a link, but for this, we might need to add the UI component on Engine Dashboard to make CRUD IP Allowlist easily like we have for CORS.

example: Unauthorized IP Address. See: <link_to_engine_dashboard>

Reply:
> We can make the message more helpful by adding a link, but for this, we might need to add the UI component on Engine Dashboard to make CRUD IP Allowlist easily like we have for CORS.
> 
> example: Unauthorized IP Address. See: <link_to_engine_dashboard>

I agree!
Do I work on the dashboard changes too, then put the link here, and we release a new version of engine and dashboard simultaneously? Or maybe I just put in the "anticipated" URL here now?